packages/ui/src/stories/formkit/Color.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
<img width="229" alt="image" src="https://github.com/user-attachments/assets/47bfa2ab-dfe8-40da-a5da-10dd7ffa2400" />

This doesn't look ok to me.